src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isHusband = sex === 'f' ? 'husband' : 'wife';
```

Comment:
The logic for determining `isHusband` is incorrect. It should be `sex === 'm' ? 'wife' : 'husband'` to correctly assign 'husband' when the person is female and 'wife' when the person is male.